code_challenges.rb

```diff
@@ -21,6 +32,25 @@
 # Expected output: ['tea', 'soda water']
 
 
+def lett_includ (array, letter)
+    array.select do |value|
+        if value.include? letter
+        value
+        end
+    end    
+end  
+
+# def lett_includ array
+#     new_arr = array. do |value|
+        
+#     end
+#      new_arr  
+# end  
+
+p '2.1'
+p lett_includ beverages_array, letter_o
+p '2.2'
+p lett_includ beverages_array, letter_a
 
```

Comment:
Logic looks great. Because you are using a select statement, you don't have to use an if block. Select offers you a built in if statement just like filter.